x-pack/plugins/fleet/server/services/epm/fields/field.ts

```diff
@@ -246,8 +246,23 @@ export const getField = (fields: Fields, pathNames: string[]): Field | undefined
   return undefined;
 };
 
+export function processFieldsWithWildcard(fields: Fields): Fields {
+  const newFields: Fields = [];
+  for (const field of fields) {
+    const hasWildcard = field.name.includes('*');
+    const hasNotObjectType = !field.object_type;
```

Comment:
This would probably be a bit more sensible as a "positive" condition that's negated below

```ts
const hasObjectType = !!field.object_type

if (hasWildcard && !hasObjectType) ...
```

Reply:
yes it make sense, having a bad time naming variables on this one